juju-charms/sample_overlay_bundles/bionic-queens-nfs.yaml

```diff
@@ -0,0 +1,96 @@
+variables:
+  openstack-origin: &openstack-origin cloud:bionic-train
+  triliovault-pkg-source: &pkg-source "deb [trusted=yes] https://apt.fury.io/triliodata-4-2/ /"
+
+series: bionic
+applications:
+  trilio-data-mover:
+    charm: trilio-data-mover
+    channel: 4.2/stable
+    options:
+       triliovault-pkg-source: *pkg-source
+       nfs-shares: <NFS server IP>:<NFS share path>
+       nfs-options: nolock,soft,timeo=180,intr,lookupcache=none,nfsvers=3
+       verbose: true
```

Comment:
verbose is not set to true for all the trilio components for all the bundles, please check and add wherever it can be added.

Reply:
Fixed